src/main/java/ro/societateahermes/backendservice/utils/mapper/SponsorMapper.java

```diff
@@ -0,0 +1,39 @@
+package ro.societateahermes.backendservice.utils.mapper;
+
+import ro.societateahermes.backendservice.entities.Sponsor;
+import ro.societateahermes.backendservice.entities.dto.SponsorDTO;
+import ro.societateahermes.backendservice.repository.EventRepositoryInterface;
+
+
+public class SponsorMapper {
+
+    private EventRepositoryInterface eventRepository;
```

Comment:
codul tau este 99% excelent, dar la ce ma rog dai un repo catre un mapper? :))

Reply:
In dto tin doar idEvent iar in entitate am eventul si altfel nu am stiut cum sa fac maparea, asa fac un getOne.....nu imi place ca am repo in mapper dar varianta asta am gasit-o:)))

Reply:
ok, are sens atunci. No ok lasa asa :))

Reply:
de fapt mi-a venit o idee. adnoteaza fieldul de repo cu autowired

Reply:
problema rezolvata